test/viewport.test.js

```diff
@@ -1,11 +1,30 @@
-/*global describe, it*/
+/*global describe, it, before, after*/
 'use strict';
 
 var expect = require('expect.js');
 
 var oViewport = require('./../main.js');
 
+function isPhantom(){
+	return /PhantomJS/.test(navigator.userAgent);
+}
+
 describe('o-viewport', function() {
+	before(function(){
+		if(isPhantom()) {
+			// hack to make test run in PhantomJS
+			// it doesn't support visibilitychange
```

Comment:
Maybe add support to it to the polyfill service and document it in the origami.josn